### DISCUSSION TITLE
Carelink xdrip active insulin problem

### DISCUSSION THREAD

**Discussion by raffym66** [NightscoutFoundation/xDrip#3254] [Bug reports]
Greetings, after the version dated 14.11.2023 all the released versions do not have the hourglass with the active insulin count, why not?
For data I use carelink
![Screenshot_2023-12-25-08-38-42-886_com eveningoutpost dexdrip-edit](https://github.com/NightscoutFoundation/xDrip/assets/75333092/f6cebdfb-4c79-4a8f-8d82-4bfc99183623)
![Screenshot_2023-12-25-08-38-56-120_com eveningoutpost dexdrip-edit](https://github.com/NightscoutFoundation/xDrip/assets/75333092/9acca130-de5d-446b-ab26-a083ddf12e86)

**Comment by violinjon** [comment]
Hello, I have the same issue, although for me the last correctly working nightly is the Dec 05 2023. All the ones after that don't show active insulin.

Using CareLink Follower to follow my wife's 780G account.

Would also be great if the sensor expiration info could show like as seen in the docs, this has never worked for me. I assume something changed with CareLink last year.

**Reply by Navid200** [reply]
You are making a reference to CareLink not the person who started this thread.

If this is about CareLink, I can ask the person who knows about it to help.
But, if a thread is started with no details, I have no idea what it is about and who may be able to help.

**Reply by violinjon** [reply]
The OP is also using CareLink and appears to have a similar issue. The data in question is from the Pump Status field in the Extra Lines settings, data supplied by the CareLink data source. I posted on a FB user group and someone suggested I respond to this post. 

Please let me know what details you need for this to be helpful, thanks!

**Comment by Navid200** [comment]
@benceszasz Can you think of what could have caused this?

**Reply by benceszasz** [reply]
@Navid200 I will look into it.

**Comment by raffym66** [comment]
HI, I'm using the November version to see the active insulin, for the
expiration of the sensor and the sets I'm writing everything down because
my daughter is 12 years old and forgets everything.

Il sab 13 gen 2024, 02:14 violinjon ***@***.***> ha scritto:

**Reply by Navid200** [reply]
Please edit the title and include CareLink in it.
xDrip can be used with so many different data sources.

**Comment by raffym66** [comment]
what does carelink have to do with it, I'm just wondering why xdrip no
longer records active insulin data.

Il sab 13 gen 2024, 14:19 Navid ***@***.***> ha scritto:

**Reply by Navid200** [reply]
Why do you think xDrip no longer records active insulin data?
Have you tested xDrip with a hardware data source other than CareLink?
If you have, you should include the details about your tests here.

**Comment by raffym66** [comment]
the hourglass with the data no longer comes out and I can only use the data
from the carelink server my daughter has Medtronic 780g with G4 sensor

Il sab 13 gen 2024, 14:29 Navid ***@***.***> ha scritto:

**Reply by Navid200** [reply]
Hence my suggestion to edit the title.  I'm not sure what you are getting at.
I have already asked someone who added CareLink to xDrip to have a look at this.  You now have to be patient and wait for him.
I would have asked 3 weeks ago when you posted here if you had clarified you used CareLink.

**Comment by raffym66** [comment]
Ok

Il sab 13 gen 2024, 14:37 Navid ***@***.***> ha scritto:

**Comment by raffym66** [comment]
Last version gen 2024 not present active insuline
![Screenshot_2024-01-16-23-06-22-780_com eveningoutpost dexdrip](https://github.com/NightscoutFoundation/xDrip/assets/75333092/41da82d4-4e68-470b-8cf6-1367e62050df)

**Reply by raffym66** [reply]
No response ?
